contracts/src/v0.8/keystone/CapabilityRegistry.sol

```diff
@@ -447,13 +463,10 @@ contract CapabilityRegistry is OwnerIsCreator, TypeAndVersionInterface {
 
       Node storage node = s_nodes[p2pId];
 
-      bool nodeExists = bytes32(node.signer) != bytes32("");
-      if (!nodeExists) revert InvalidNodeP2PId(p2pId);
+      if (bytes32(node.signer) == bytes32("")) revert InvalidNodeP2PId(p2pId);
```

Comment:
```suggestion
      if (bytes32(node.signer) == bytes32("")) revert ???(p2pId);
```
The error message here is misleading.

Reply:
Shouldn't this check p2pId?

Reply:
I was thinking it could be either.  We are basically just checking to see if the `p2pId` on line 464 queried a valid node.  Wdyt about changing this error message to be `NodeDoesNotExist` to match the other error messages when an entity does not exist?